pkgs/development/python-modules/globre/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv
+, pythonAtLeast
+, buildPythonPackage
+, fetchPypi
+, nose
+, coverage
+}:
+buildPythonPackage rec {
+  pname = "globre";
+  version = "0.1.5";
+  # https://github.com/metagriffin/globre/issues/7
+  disabled = pythonAtLeast "3.7";
```

Comment:
probably same comment here

Reply:
unfortunately this one's right

Reply:
this is unfortunate :(

Reply:
it is. Could alternatively skip the test suite, which I'd personally be fine with (this is a transitive dependency 3 levels deep for me)

Reply:
Generally test should only be disabled if they are incompatible with the sandbox environment. It looks like the tests were asserting that some assumptions were broken by a new interpreter.

Reply:
yeah this is actually a case of a broken test suite, rather than an impure one

Reply:
fwiw this PR is still useful to me, as we're still on python 3.6